Toolbar.php

```diff
@@ -32,14 +39,33 @@ public function __construct()
         $renderer->disableVendor('jquery');
         $renderer->disableVendor('fontawesome');
         $renderer->setUseRequireJs(true);
+
+        $this->setStorage($storage);
+        $renderer->setOpenHandlerUrl('/hackathon_toolbar/openhandler/handle');
```

Comment:
Please inject \Magento\Store\Model\StoreManagerInterface and use $renderer->setOpenHandlerUrl($storeManager->getStore()->getBaseUrl() . 'hackathon_toolbar/openhandler/handle');

\Magento\Framework\UrlInterface is the more correct way of getting an URL but in this stage of the request the area code is not set yet